src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -26,7 +26,58 @@ def setUp(self):
 
     def tearDown(self):
         pass
-		
+    
+    def assertCbsdRecord(self, registration_request, grant_request, grant_response, recordData):
+        for index, device in enumerate(registration_request):
+            record_id = 'cbsd/'+ device['fccId']+'/'+ hashlib.sha1(device['cbsdSerialNumber']).hexdigest()
+            cbsd_record = [record['registration'] for record in recordData if record['id'] == record_id]
+            self.assertEqual(1, len(cbsd_record))
+            self.assertEqual(device['cbsdCategory'], cbsd_record[0]['cbsdCategory'])
+            self.assertEqual(device['fccId'], cbsd_record[0]['fccId'])
+            self.assertEqual(device['callSign'], cbsd_record[0]['callSign'])
```

Comment:
callSign is an optional field in SAS-SAS exchange. *If* present, it should be equal to the value given in the registration request... but it can also be missing.